mevolib/fetch/BioSeqs.py

```diff
@@ -529,4 +530,23 @@ def statistics (self):
         std_value = numpy.std(seqs_length)
         min_value = numpy.amin(seqs_length)
         max_value = numpy.amax(seqs_length)
-        return (len(self), mean_value, std_value, min_value, max_value)
\ No newline at end of file
+        return (len(self), mean_value, std_value, min_value, max_value)
+
+
+def call_fetch_gb_seqs(query: str, name: str):
+    """docstring"""
+    seq_db = BioSeqs.from_entrez(
+        email="ex@ucm.es", entrez_db="nuccore",
+        query = query, max_fetch=10
```

Comment:
```suggestion
        email="ex@ucm.es", entrez_db="nuccore", query=query, max_fetch=10
```
Furthermore, the email should not be hardcoded (specially not an UCM one!).